packages/flutter/lib/src/material/card.dart

```diff
@@ -79,7 +79,9 @@ class Card extends StatelessWidget {
   /// Defines the card's [Material.color].
   ///
   /// If this property is null then [CardTheme.color] of [ThemeData.cardTheme]
-  /// is used. If that's null then [ThemeData.cardColor] is used.
+  /// is used. If that's null, then [ThemeData.cardColor] is used when material2
+  /// is enabled, [ColorScheme.surface] of [ThemeData.colorScheme] is used when
+  /// material 3 enabled.
```

Comment:
Looks good, but is slightly incorrect (it was before as well). The card theme actually comes from `CardTheme.of` which can default to `ThemeData.cardTheme`, but the widget might be surrounded by a `CardTheme` widget. II would replace the whole paragraph with something like:

```suggestion
  /// If this property is null then the ambient [CardTheme.color] is used. If that is null,
  /// and [ThemeData.useMaterial3] is true, then [ColorScheme.surface] of 
  /// [ThemeData.colorScheme] is used. Otherwise, [ThemeData.cardColor] is used.
  ```